### DISCUSSION TITLE
issue commites

### DISCUSSION THREAD

**Discussion by eyrexh** [UBC-MDS/chocolate_rating#66] [Show and tell]
- Explain why the MAPE score is used as the metric for predictions from @austin-shih and @dhruvinishar feedback: https://github.com/UBC-MDS/chocolate_rating/commit/a09a39137c81bbe284c13a7e68024a967f241181
- Round the numeric values in report tables and inline in text from @spencergerlach feedback: https://github.com/UBC-MDS/chocolate_rating/commit/e5e869dcfebfa928c31c2c4cbfd8cfac2e1f21e4
- The scripts could use fewer options from @ataciuk feedback: https://github.com/UBC-MDS/chocolate_rating/commit/49e1d17bc2660b138e882b8f02c161074d69887f
- Add tests for your code from @dhruvinishar feedback: https://github.com/UBC-MDS/chocolate_rating/commit/885ca6671b756f93e1f5411e1d02a0536ea6d89d
- Add the report as a markdown file from @ataciuk feedback: https://github.com/UBC-MDS/chocolate_rating/commit/00a82652957b8e8f890d0fdbca716db171ad5ab0

**Comment by robindhillon1** [comment]
- (Peer Review) Fixed the formatting as well as the commands suggested by @ataciuk via peer review. https://github.com/UBC-MDS/chocolate_rating/commit/4e5093a64ad9f46dfd857006843851e459ec041e
- (Peer Review + Milestone 1 Feedback): Fixed ambiguous wording relating to the research question mentioned by the TA and peer review. Now the research question is clear: https://github.com/UBC-MDS/chocolate_rating/commit/d54891d2281328a8ec25bf41058478fdeea3fe2d
- (Milestone 2): Added `kableExtra` in the installation section, along with other packages: https://github.com/UBC-MDS/chocolate_rating/commit/9a9b8748070f6a80ece363ae854634ce1b90d0e4
- (Milestone 1): Clarity of readme has been improved to reduce grammatical errors and other useful information.
    -  https://github.com/UBC-MDS/chocolate_rating/commit/4e5093a64ad9f46dfd857006843851e459ec041e
    -  https://github.com/UBC-MDS/chocolate_rating/commit/554ca8a1d75dccb2cc58de5d2816994b7ab272b1
    -  https://github.com/UBC-MDS/chocolate_rating/commit/9787bf5b603163b65a76d847594139f630d9dafe
- Added `docopt` to dependencies: https://github.com/UBC-MDS/chocolate_rating/commit/48137d6110e59bf657e348a33f921a8d09bc784d
- Added `pandoc` according to TA feedback: https://github.com/UBC-MDS/chocolate_rating/commit/599b09c18cfaab55f36e0145c5b828bf74599a30
- Fixed contributing file feedback from Milestone 1. Added core team member contribution: https://github.com/UBC-MDS/chocolate_rating/commit/6ac09e4902213053de9a3819d020dd50a3c55955
- Made the consequences clearer; very minor update since everything seems correct already: https://github.com/UBC-MDS/chocolate_rating/commit/7d6aa598e23ae511b800a12f2fa8dc4d75c2382d

**Comment by lieeeee** [comment]
您好，我是李鹏飞，您的邮件已收到，我会尽快处理。

**Comment by ArmstrongYang** [comment]
您好，已经收到邮件，我会尽快处理给予回复，谢谢！
